### PR TITLE
DEV | Refactor user models

### PR DESCRIPTION
**Pull request by ceydaduzgec**
- Delete Profile model
- Delete Contact model

### CODE REVIEW

File: users/templates/my_account.html [hunk 1]
```diff
@@ -64,25 +64,12 @@ <h1>Edit your account</h1>
                     <form method="POST" enctype="multipart/form-data">
                         {% csrf_token %}
                         {#{{ user_form.as_p }}#}
-                        {#{{ profile_form.as_p }}#}
+                        {% for field in user_form %}
```
Comment (ceydaduzgec) [inline users/templates/my_account.html:67]: @SahinKapan We shouldn't type in the form fields statically, they should be dynamic and can be controlled from the backend without needing to changing html files. Can you please refactor the forms like this? I can help you with that if you need.

**Comment by ceydaduzgec** [comment]
We can take a look at this after the milestone, I don't want to broke some things before it.

File: spaces/urls.py [hunk 2]
```diff
@@ -20,4 +21,5 @@
     path("spaces/<int:pk>/members/", space_members, name="space_members"),
     path("spaces/<int:pk>/join/", join_space, name="join_space"),
     path("spaces/<int:pk>/leave/", leave_space, name="leave_space"),
+    path("newspace/", newspace, name="newspace"),
```
Comment (ceydaduzgec) [inline spaces/urls.py:24]: @MariaAsif135 I am moving this to spaces app since it is related with that.

**Review by sehmuzc — APPROVED** [review]
Thanks for the refactorings.

File: spaces/urls.py (hunk 2 above)
Comment (sehmuzc) [reply spaces/urls.py:24]: Thanks, we need to work on the integration part of this newspace part with the space join functions.

File: users/templates/my_account.html (hunk 1 above)
Comment (sehmuzc) [reply users/templates/my_account.html:67]: @SahinKapan when you comment on this conversation and resolve conversation then the code will be eligible for merging.

**Pull request merged**